packages/plugins/response-cache-cloudflare-kv/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import type { ExecutionResult } from 'graphql';
-import type { ExecutionContext, KVNamespace } from '@cloudflare/workers-types';
+import type { ExecutionContext, KVNamespace } from '@cloudflare/workers-types/2023-07-01';
```

Comment:
Doesn't this have the potential to age poorly ? We will have to update all imports when we update the types dependency ?